src/Raven.Server/Web/System/Processors/Databases/DatabasesHandlerProcessorForGetRestorePoints.cs

```diff
@@ -61,7 +62,7 @@ protected override async ValueTask HandleCurrentNodeAsync()
             var connectionType = GetPeriodicBackupConnectionType();
             var settings = await GetSettingsAsync(context);
 
-            using var source = GetRestorePointsSource(context, connectionType, settings, out string path);
+            using var source = GetRestorePointsSource(context, connectionType, settings, out string path, RequestHandler.AbortRequestToken);
```

Comment:
wrong token here? shouldn't we have a token bound at least to server shutdown and abort request?

Reply:
done